Remotion/Data/DomainObjects/Tracing/TracingDataReader.cs

```diff
@@ -129,6 +129,11 @@ public DateTime GetDateTime (int i)
       return _dataReader.GetDateTime(i);
     }
 
+    public DateOnly GetDateOnly (int i)
```

Comment:
This shouldn't be on TracingDataReader. This class only implements the interface and doesn't have special reader / conversion features. It's purpose is tracing. Please drop and move logic to usage site, if there is any.